src/components/pages/DownloadPage.jsx

```diff
@@ -1,31 +1,36 @@
 import Instructions from 'components/elements/Instructions'
-import { DOWNLOAD_URL } from 'constants/urls'
+import InstructionsUninstall from 'components/elements/InstructionsUninstall'
+import { DOWNLOAD_URL } from 'constants/urls';
+import downloadButton from "../../images/download-arrow.svg";
+
 
 function DownloadPage() {
   return (
     <main>
-      <div>
-        <h1>Getting the App</h1>
-        <a href={DOWNLOAD_URL}>
-          Download for Mac
-        </a>
+      <div className="flex flex-col items-center  mt-40">
+        <button className="invisible lg:visible w-350 p-4 rounded bg-[#37770D] border-black border-2 flex">
```

Comment:
Instead of visible and invisible, let's use hidden and lg:block (flex),  also let's move the classNames to be in the order of
display properties, margins/paddings, colors, height, width

so in this case:
"hidden lg:flex p-4 rounded border-black border-2 bg-[#37770D] w-350"

This way code reviewers can quickly see the display properties and there's a consistency to this throughout the app

Reply:
Changes made. Also reordered classNames in all relevant components.